pkg/i2gw/ingress2gateway.go

```diff
@@ -101,7 +56,7 @@ func ingresses2GatewaysAndHTTPRoutes(ingresses []networkingv1.Ingress) ([]gatewa
 	return aggregator.toHTTPRoutesAndGateways()
 }
 
-func outputResult(printer printers.ResourcePrinter, httpRoutes []gatewayv1beta1.HTTPRoute, gateways []gatewayv1beta1.Gateway) {
+func OutputResult(printer printers.ResourcePrinter, httpRoutes []gatewayv1beta1.HTTPRoute, gateways []gatewayv1beta1.Gateway) {
```

Comment:
this feels print related. so I would turn this into `func (pr *printRunner) outputResult()`